src/rprblender/nodes/blender_nodes.py

```diff
@@ -410,20 +410,28 @@ class ShaderNodeBsdfVelvet(RuleNodeParser):
     # inputs: Color, Sigma
 
     nodes = {
+        "ONE_MINUS_SIGMA": {
+            "type": "-",
+            "params": {
+                pyrpr.MATERIAL_INPUT_COLOR0: 1.0,
+                pyrpr.MATERIAL_INPUT_COLOR1: "inputs.Sigma",
+            }
+        },
+
         "BSDF": {
             "type": pyrpr.MATERIAL_NODE_UBERV2,
             "params": {
                 pyrpr.MATERIAL_INPUT_UBER_DIFFUSE_COLOR: "inputs.Color",
-                pyrpr.MATERIAL_INPUT_UBER_DIFFUSE_WEIGHT: "inputs.Sigma",
+                pyrpr.MATERIAL_INPUT_UBER_DIFFUSE_WEIGHT: "nodes.ONE_MINUS_SIGMA",
```

Comment:
Sigma can be set to < 0.0. The Cycles result is just black if sigma 0.0 and less, this change makes RPR way too bright.

Reply:
I would claim those are invalid values.  At any rate.  My goal here was to make BMW floor make good.

Reply:
Cycles does clamp(0.0, 1.0) on Sigma input value. I think so should we.